components/src/androidTest/java/uk/gov/android/ui/components/m3/ButtonsM3Test.kt

```diff
@@ -33,12 +29,29 @@ class ButtonsM3Test(
         Assert.assertEquals(
             "The expected size of the provider has changed!",
             expectedParameterSize,
-            values().size
+            parametersList.size
         )
     }
 
     @Test
-    fun buttonTests() {
+    fun buttonTest1() = buttonTest(parametersList[0])
+
+    @Test
+    fun buttonTest2() = buttonTest(parametersList[1])
+
+    @Test
+    fun buttonTest3() = buttonTest(parametersList[2])
+
+    @Test
+    fun buttonTest4() = buttonTest(parametersList[3])
+
+    @Test
+    fun buttonTest5() = buttonTest(parametersList[4])
+
+    @Test
+    fun buttonTest6() = buttonTest(parametersList[5])
+
```

Comment:
I think there might be more than 6 parameters - if we add the rest would that help increase the test coverage maybe?